.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
This appeared after setting the eslint configuration in IntelliJ to automatic, I guess we want this project wide then?